src/pages/Start.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState, useEffect } from "react";
+import { Navigate } from "react-router-dom";
+import appicon from "../data/appicon.png";
+
+const Start = () => {
```

Comment:
Not a code, but product comment. I thought this start screen was a bit redundant :D!